src/WebJobs.Script/Diagnostics/Extensions/ScriptHostLoggerExtension.cs

```diff
@@ -12,111 +12,123 @@ internal static class ScriptHostLoggerExtension
-           "Adding Function descriptor provider for HttpWorker.");
+               LogLevel.Debug,
+               new EventId(414, nameof(AddingDescriptorProviderForHttpWorker)),
+               "Adding Function descriptor provider for HttpWorker.");
 
         private static readonly Action<ILogger, string, Exception> _stoppingScriptHost =
             LoggerMessage.Define<string>(
-            LogLevel.Debug,
-            new EventId(415, nameof(StoppingScriptHost)),
-            "Stopping ScriptHost instance '{hostInstanceId}'.");
+                LogLevel.Debug,
+                new EventId(415, nameof(StoppingScriptHost)),
+                "Stopping ScriptHost instance '{hostInstanceId}'.");
 
         private static readonly Action<ILogger, string, Exception> _stoppedScriptHost =
-          LoggerMessage.Define<string>(
-          LogLevel.Debug,
-          new EventId(416, nameof(StoppedScriptHost)),
-          "Stopped ScriptHost instance '{hostInstanceId}'.");
+            LoggerMessage.Define<string>(
+                LogLevel.Debug,
+                new EventId(416, nameof(StoppedScriptHost)),
+                "Stopped ScriptHost instance '{hostInstanceId}'.");
 
         private static readonly Action<ILogger, string, Exception> _disposingScriptHost =
-          LoggerMessage.Define<string>(
-          LogLevel.Debug,
-          new EventId(417, nameof(DisposingScriptHost)),
-          "Disposing ScriptHost instance '{hostInstanceId}'.");
+            LoggerMessage.Define<string>(
+                LogLevel.Debug,
+                new EventId(417, nameof(DisposingScriptHost)),
+                "Disposing ScriptHost instance '{hostInstanceId}'.");
 
         private static readonly Action<ILogger, string, Exception> _disposedScriptHost =
-          LoggerMessage.Define<string>(
-          LogLevel.Debug,
-          new EventId(418, nameof(DisposedScriptHost)),
-          "Disposed ScriptHost instance '{hostInstanceId}'.");
+            LoggerMessage.Define<string>(
+              LogLevel.Debug,
+              new EventId(418, nameof(DisposedScriptHost)),
+              "Disposed ScriptHost instance '{hostInstanceId}'.");
+
+        private static readonly Action<ILogger, string, Exception> _functionsWorkerRuntimeValue =
+            LoggerMessage.Define<string>(
+                LogLevel.Debug,
+                new EventId(419, nameof(FunctionsWorkerRuntimeValue)),
+                $"{EnvironmentSettingNames.FunctionWorkerRuntime} value: '{{workerRuntime}}'");
+
+        private static readonly Action<ILogger, string, Exception> _resolvedWorkerRuntimeFromMetadata =
+            LoggerMessage.Define<string>(
+                LogLevel.Debug,
+                new EventId(420, nameof(ResolvedWorkerRuntimeFromMetadata)),
+                $"{EnvironmentSettingNames.FunctionWorkerRuntime} is null. Resolved worker runtime from function metadata: '{{workerRuntime}}'");
```

Comment:
Did you mean to have two `{{` ? I see all other entries not doing that.

Reply:
yeah -- it's b/c this one is using string interpolation (`$`). So need to escape the `{}` around workerRuntime. It ends up being `FUNCTIONS_WORKER_RUNTIME is null. Resolved worker Resolved worker runtime from function metadata: '{workerRuntime}'`